src/components/search/Search.js

```diff
@@ -1,14 +1,81 @@
-import { SearchAlt2 } from '@styled-icons/boxicons-regular'
+import '@reach/combobox/styles.css'
+
+import {
+  Combobox,
+  ComboboxInput,
+  ComboboxList,
+  ComboboxOption,
+} from '@reach/combobox'
+import { useHistory } from 'react-router-dom'
+import usePlacesAutocomplete, {
+  getGeocode,
+  getLatLng,
+} from 'use-places-autocomplete'
+import { NumberParam, useQueryParams } from 'use-query-params'
 
 import Input from '../ui/Input'
+import SearchEntry from './SearchEntry'
+
+const Search = () => {
+  const [centerCoords, setCenterCoords] = useQueryParams({
+    centerLat: NumberParam,
+    centerLng: NumberParam,
+  })
```

Comment:
Make sure this eventually matches with the format on view in MapPage. You should only need one `useQueryParams` call. Also, see if this type is useful:

![image](https://user-images.githubusercontent.com/4369024/110537681-9d9cd080-80e8-11eb-8223-f962c47fcbd3.png)

https://github.com/pbeshai/use-query-params#param-types

Reply:
`const [view, setView] = useQueryParams(...)`